README.md

```diff
@@ -25,28 +20,64 @@ Get it and use it:
     git clone https://github.com/multipath-tcp/mptcp-vagrant.git
     cd mptcp-vagrant
     # only the first time:
-    vagrant plugin install vagrant-triggers
+    git submodule update --init --recursive
     vagrant up
+    ./scripts/init-mptcp.sh
 
 This will:
 
   * download a vagrant box
   * start the virtual machine
-  * setup NAT
-
-To validate all works as expected, issue this command:
+  
+You can then connect to the client/server using
 
-    vagrant ssh -c "curl www.multipath-tcp.org"
+    vagrant ssh (client | server)
 
-The outpout should be message full of joy, congratulating you for your MPTCP capabilities!
 
 You stop the vm by issuing
 
     vagrant halt
 
-This will also remove the NAT that was setup when starting the vm.
+Measuring the traffic
+=====================
+
+You can use `pmrep` to check the traffic generated on the different interface.
+This can be typically useful if you want to see if data is well sent in parallel
+on multiple interfaces (example: if you are running `mptcpize run iperf3`).
+
+```sh
+# Check traffic for 10 seconds, every second
+pmrep network.interface.in.bytes -t 1 -s 10
+```
+
+Here is an example of output generated by pmrep while an iperf3 was running:
+
+```
+  n.i.i.bytes  n.i.i.bytes  n.i.i.bytes  n.i.i.bytes  n.i.i.bytes
+           lo         eth0         eth1         eth2         eth3
+       byte/s       byte/s       byte/s       byte/s       byte/s
+          N/A          N/A          N/A          N/A          N/A
+        0.000     2973.340        0.000        0.000        0.000
+        0.000      751.777    35785.253    29314.985    25192.125
+        0.000        0.000    65085.137    48311.050    49763.924
+        0.000        0.000    75632.805    44275.642    54212.010
+        0.000        0.000    71197.406    53428.050    51910.276
+        0.000        0.000    75361.548    48189.643    44524.735
+        0.000        0.000    63875.409    44105.733    55144.152
+        0.000        0.000    64017.342    46963.665    46195.048
+        0.000        0.000    66311.320    54552.121    43777.357
+        0.000        0.000    58849.307    52688.773    52658.751
+        0.000        0.000    69949.390    40244.443    45697.149
+        0.000        0.000    76594.098    50984.709    39281.333
+        0.000        0.000    60517.017    42658.258    57715.525
+        0.000        0.000    58039.906    52795.697    48618.310
+        0.000       60.015    75516.635    44548.993    47093.621
+        0.000        0.000    74750.513    49240.132    36631.828
+        0.000      452.748    32816.216    20149.289    26109.135
+        0.000        0.000        0.000        0.000        0.000
+        0.000        0.000        0.000        0.000        0.000
+``` 
   
-
 Thanks
```

Comment:
It looks like the NAT is no longer needed. Feel free to remove this section if that's the case.